x-pack/plugins/ml/public/application/datavisualizer/file_based/components/file_datavisualizer_view/file_datavisualizer_view.js

```diff
@@ -22,12 +22,12 @@ import { ExplanationFlyout } from '../explanation_flyout';
 import { ImportView } from '../import_view';
 import {
   DEFAULT_LINES_TO_SAMPLE,
-  getMaxBytes,
   readFile,
   createUrlOverrides,
   processResults,
   hasImportPermission,
 } from '../utils';
+import { getFileUpload } from '../../../../util/dependency_cache';
```

Comment:
it's probably fine to use `getFileUpload` here as this file will be rewritten soon